src/include/parser/parser.h

```diff
@@ -73,10 +73,10 @@ typedef enum
 extern PGDLLIMPORT int backslash_quote;
 extern PGDLLIMPORT bool escape_string_warning;
 extern PGDLLIMPORT bool standard_conforming_strings;
-extern int sql_dialect;
-extern bool pltsql_case_insensitive_identifiers;
+extern PGDLLEXPORT int sql_dialect;
+extern PGDLLEXPORT bool pltsql_case_insensitive_identifiers;
 
-extern char* pltsql_server_collation_name;
+extern PGDLLEXPORT char* pltsql_server_collation_name;
```

Comment:
This is the only Babelfish-specific change made for this series of commits: changing these three extern variables to use PGDLLEXPORT so that they stay visible for the extension to consume.